server/models/PaymentMethod.js

```diff
@@ -409,7 +409,7 @@ function defineModel() {
 
     // Independently of the balance of the external source, the owner of the payment method
     // may have set up a monthlyLimitPerMember or an initialBalance
-    if (!this.initialBalance && (!this.monthlyLimitPerMember || (user && user.isAdmin(this.CollectiveId)))) {
+    if (!this.initialBalance && !this.monthlyLimitPerMember) {
```

Comment:
I changed this behavior because, after changing the role in `test/server/graphql/v1/createOrder.test.js` to make sure the `createOrder` was tested with an `ADMIN`, the test failed as the expected balance went to "MAX_INTEGER".

It seems that, in this portion of the code, the `monthlyLimitPerMember` was only enforced for `MEMBER`, not `ADMIN`. While this was certainly the expected behavior in the initial implementation, it does not match the current one.